packages/@o3r/test-helpers/src/prepare-test-env.ts

```diff
@@ -89,7 +89,11 @@ export async function prepareTestEnv(folderName: string, options?: PrepareTestEn
   const prepareFinalApp = (baseApp: string) => {
     logger.debug?.(`Copying ${baseApp} to ${workspacePath}`);
     const baseProjectPath = path.join(itTestsFolderPath, baseApp);
-    cpSync(baseProjectPath, workspacePath, { recursive: true, dereference: true, filter: (source) => !/node_modules/.test(source) });
+    cpSync(baseProjectPath, workspacePath, {
+      recursive: true,
+      dereference: true,
+      filter: (source) => !/node_modules/.test(source) && !/.git[\\/]/.test(source)
```

Comment:
```suggestion
      filter: (source) => !/node_modules/.test(source) && !/\\.git[\\/]/.test(source)
```
to avoid le path like `digit/` :D

Reply:
legit